uint32               gloveID
float32              timestemp
SensorDataSingle[18] sensor_data